app/src/components/governance/PageProposal.vue

```diff
@@ -19,7 +19,14 @@
 
       <ParticipantList v-if="participants" :participants="participants" />
 
-      <template v-if="proposal.detailedVotes.timeline">
+      <template
+        v-if="
+          proposal.detailedVotes.timeline &&
+          Object.values(proposal.detailedVotes.timeline).find(
```

Comment:
why do we need to check if they have `time` entries?

Reply:
Because if `time` is `null` you get an `Invalid Date` message

Reply:
if `time` is null, then what is in `timeline`?

Reply:
change the API not to include timeline objects if they don't have time